src/Instagram/Provider.php

```diff
@@ -2,23 +2,32 @@
 
 namespace SocialiteProviders\Instagram;
 
-use GuzzleHttp\RequestOptions;
 use SocialiteProviders\Manager\OAuth2\AbstractProvider;
 use SocialiteProviders\Manager\OAuth2\User;
 
 class Provider extends AbstractProvider
 {
+    /**
+     * Unique Provider Identifier.
+     */
     public const IDENTIFIER = 'INSTAGRAM';
 
     /**
      * {@inheritdoc}
      */
     protected $scopeSeparator = ' ';
 
+    /**
+     * The user fields being requested.
+     *
+     * @var array
+     */
+    protected $fields = ['account_type', 'id', 'username', 'media_count', 'profile_picture'];
```

Comment:
Do these need to be customisable?

Reply:
No, those are the only fields available. https://developers.facebook.com/docs/instagram-basic-display-api/reference/user#fields